debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -142,7 +142,7 @@ public void changeRecord(DataCollectionSchema schema, Operation operation, Objec
 
     public void dispatchSchemaChangeEvent(T dataCollectionId, SchemaChangeEventEmitter schemaChangeEventEmitter) throws InterruptedException {
         if(!filter.isIncluded(dataCollectionId)) {
-            LOGGER.trace("Skipping data change event for {}", dataCollectionId);
+            LOGGER.trace("Filtering schema change event for {}", dataCollectionId);
```

Comment:
@jpechane I dropped the metric update and fixed the log statement to denote it being a schema change event.